public/swagger.yaml

```diff
@@ -36,7 +46,6 @@ paths:
           examples:
             application/json: { "success": "true", "message": "Server is up!", "status": "UP" }
 
```

Comment:
Realized we don't have the ping route in the swagger so the route wasn't registered:

```
/ping:
    x-swagger-router-controller: ping
    get:
      operationId: pingMe
      tags:
        - Ping
      description: Ping blockchain network
      responses:
        200:
          description: Ping response
          schema:
            $ref: "#/definitions/ResponseBody"
```